app/models/mixins/authentication_mixin.rb

```diff
@@ -288,7 +288,6 @@ def authentication_check_no_validation(type, options)
           [:error, err]
         end
       end
-
```

Comment:
Well if there was no index.html.haml the problem was that it threw warning in console, so I solved it by empty index.html
Not sure how else could this be solved, since I am new to ruby.